src/chocolatey.console/Program.cs

```diff
@@ -370,5 +374,24 @@ private static void write_warning_for_incompatible_versions()
 Or by passing the --skip-compatibility-checks option when executing a
 command.");
         }
+
+        private static void check_installed_dotnetfx_version()
+        {
+            if (Platform.get_platform() == PlatformType.Windows)

```

Comment:
Do we need to worry about execution on an earlier version of Mono?

Reply:
Maybe. 

Mono 6.6 is the first version with the .NET 4.8 assemblies, so anything earlier than that may not work.

However, the only recommended current way to run choco on non-Windows is in the docker container, and that has a new enough version of Mono. 

It is possible to get the display version of mono via reflection, but I don't see a great way to parse it that would work for versions from custom/git builds of mono and all of the various possible OS names. 
https://stackoverflow.com/questions/4178129/how-to-determine-the-revision-from-which-current-mono-runtime-was-built-and-inst

Reply:
This also makes sense. Nothing else needs to be done here.